### PR TITLE
Sample CCS integration document

### PR DESCRIPTION
**Pull request by danielsalzer42**

### CODE REVIEW

File: ccs/interviews.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+* [Get Interview](#get-interviewsinterviewid)
+* [Get Interviews](#get-interviews)
+* [Create Interview](#post-interview)
+* [Update Interview](#put-interviewsinterviewid)
+* [Delete Interview](#delete-interviewsinterviewid)
+
+---
+
+### `GET` `/interviews/:id`
```
Comment (yoloVoe) [inline ccs/interviews.md]: The link in line 3 of this file doesn't currently work.
The following update should make it work.
```suggestion
### `GET` `/interviews/:interviewId`
```

File: ccs/interviews.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+  "reportUrl": "www.acmecorp.com/interviews/2902c52a-d45b-4f0f-88f0-94351c07095f/report"
+}
+```
+
+---
+
+
+
+
+### `PUT` `/interviews/:id`
```
Comment (yoloVoe) [inline ccs/interviews.md]: Link issue mentioned above
```suggestion
### `PUT` `/interviews/:interviewId`
```

File: ccs/interviews.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+```
+
+
+---
+
+
+
+
+
+### `DELETE` `/interviews/:id`
```
Comment (yoloVoe) [inline ccs/interviews.md]: Link issue mentioned above
```suggestion
### `DELETE` `/interviews/:interviewId`
```

File: ccs/interviews.md [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+{
+  "from": "2020-02-19T11:43:31+0000",
+  "to": "2020-04-20T10:44:31+0000",
+  "title": "Interview with Samir",
+  "notes": "Assess the candidate for Javascript programming skills",
+  "resumeUrl": "https://www.linkedin.com/98646494",
+  "interviewers": [
+    "jpello@lenscorp.com"
+  ],
+  "resultUrl": "https://www.lenscorp.com/interview_ended",
```
Comment (yoloVoe) [inline ccs/interviews.md:144]: I am probably missing some context, but hat is the `resultUrl` used for?

File: ccs/interviews.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+  "candidate": {
+    "name": "Samir Sanchez",
+    "email": "ssanchez74839@gmail.com"
+  },
+  "metadata": {},
+  "reportUrl": "www.acmecorp.com/interviews/2902c52a-d45b-4f0f-88f0-94351c07095f/report"
```
Comment (yoloVoe) [inline ccs/interviews.md:179]: Likewise, what is the `reportUrl` used for?

File: ccs/interviews.md [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+
+---
+
+### `GET` `/interviews/:id`
+* Returns an interview from the CCS.
+
+####  Header
+key|value
+---|---
+`Authorization`| Token
```
Comment (yoloVoe) [inline ccs/interviews.md:17]: The docs currently say
```
`Authorization`| Basic authorization (`Basic ` + base64(`api key` + `:`))
```
in some places and
```
`Authorization`| Token
```
in other places.

Is there a reason we're using two different formats?

**Review by yoloVoe — CHANGES_REQUESTED** [review]

File: ccs/interviews.md (hunk 4 above)
Comment (danielsalzer42) [reply ccs/interviews.md:144]: resultUrl description: "Webhook url for results to be posted after interview is ended, or when thumbs/feedback is changed on a completed interview"

File: ccs/interviews.md (hunk 5 above)
Comment (danielsalzer42) [reply ccs/interviews.md:179]: reportUrl description: "Url to a report about this interview"

File: ccs/interviews.md (hunk 6 above)
Comment (danielsalzer42) [reply ccs/interviews.md:17]: good catch. Thank you, Subrat

**Pull request merged**